apps/examples/testcase/ta_tc/media/itc/itc_media_fileinputdatasource.cpp

```diff
@@ -26,23 +26,32 @@
 #define FILE_DEFAULT_RATE 16000
 #define MEDIA_FORMAT_TYPE 0
 #define COUNT 10
-
+#define BUFFER_LENGTH 32
 /****************************************************************************
  * Global Variables
  ****************************************************************************/
-static const char *file_path = MEDIA_TEST_FILE_PATH;
-static const char *test_data = "dummydata";
-static int g_flag=0;
+static const char file_path[] = "/mnt/fileinputdatasource.raw";
+static const char test_data[] = "dummydata";
 static unsigned char *buf;
+static int g_flag=0;
```

Comment:
same

Reply:
changes done and code also updated in gerrit